CHANGELOG.md

```diff
@@ -1,5 +1,34 @@
+* compute: surface error details for Compute Operation with quota exceeded errors. ([#14879](https://github.com/hashicorp/terraform-provider-google/pull/14879))
+* compute: added pathTemplateMatch and pathTemplateRewrite to routeRules in UrlMap. ([#14873](https://github.com/hashicorp/terraform-provider-google/pull/14873))
+* compute: added in ability to update Hyperdisk PD IOPS without recreation ([#14844](https://github.com/hashicorp/terraform-provider-google/pull/14844))
+* container: added `sole_tenant_config` to `node_config` in `google_container_node_pool` and `google_container_cluster` ([#14897](https://github.com/hashicorp/terraform-provider-google/pull/14897))
+* dataform: added field `workspace_compilation_overrides` to resource `google_dataform_repository` (beta) ([#14839](https://github.com/hashicorp/terraform-provider-google/pull/14839))
+* dlp: added `crypto_hash_config` to `google_data_loss_prevention_deidentify_template` ([#14870](https://github.com/hashicorp/terraform-provider-google/pull/14870))
+* dlp: added `trigger_id` field to `google_data_loss_prevention_job_trigger` ([#14892](https://github.com/hashicorp/terraform-provider-google/pull/14892))
+* dlp: added missing file types `POWERPOINT` and `EXCEL` in `inspect_job.storage_config.cloud_storage_options.file_types` enum to `google_data_loss_prevention_job_trigger` resource ([#14856](https://github.com/hashicorp/terraform-provider-google/pull/14856))
+* dlp: added multiple `sensitivity_score` field to `google_data_loss_prevention_deidentify_template` resource ([#14880](https://github.com/hashicorp/terraform-provider-google/pull/14880))
+* dlp: added multiple `sensitivity_score` field to `google_data_loss_prevention_inspect_template` resource ([#14871](https://github.com/hashicorp/terraform-provider-google/pull/14871))
+* dlp: added multiple `sensitivity_score` field to `google_data_loss_prevention_job_trigger` resource ([#14881](https://github.com/hashicorp/terraform-provider-google/pull/14881))
+* dlp: changed `inspect_template_name` field from required to optional in `google_data_loss_prevention_job_trigger` resource ([#14845](https://github.com/hashicorp/terraform-provider-google/pull/14845))
+* pubsub: allowed `definition` of `google_pubsub_schema` to change without deleting and recreating the resource by using schema revisions (https://cloud.google.com/pubsub/docs/schemas#commit-schema-revision) ([#14857](https://github.com/hashicorp/terraform-provider-google/pull/14857))
+* sql: added `POSTGRES_15` to available version docs ([#14891](https://github.com/hashicorp/terraform-provider-google/pull/14891))
+* vpcaccess: Added `connected_projects` attribute to resource `google_vpc_access_connector`. ([#14835](https://github.com/hashicorp/terraform-provider-google/pull/14835))
+
+BUG FIXES:
+* filestore: fixed an issue causing 429 errors to be retried ([#14850](https://github.com/hashicorp/terraform-provider-google/pull/14850))
```

Comment:
this should name the impacted resource

Reply:
I agree, 7-8 resources is too many to include in one note. in that case how about something like: 

```suggestion
* provider: fixed an issue on multiple resources where non-retryable quota errors were considered retryable ([#14850](https://github.com/hashicorp/terraform-provider-google/pull/14850))
```

Or you could have one note per resource - that might be better:

```suggestion
* filestore: fixed an issue on `google_filestore_RESOURCE` where non-retryable quota errors were considered retryable ([#14850](https://github.com/hashicorp/terraform-provider-google/pull/14850))
```

And then repeat that per resource